markdown.go

```diff
@@ -314,24 +319,20 @@ func MarkdownBasic(input []byte) []byte {
 //
 // * Custom Heading IDs
 func MarkdownCommon(input []byte) []byte {
-	// set up the HTML renderer
-	renderer := NewHTMLRenderer(HTMLRendererParameters{
-		Flags: CommonHTMLFlags,
-	})
-	return Markdown(input, renderer, DefaultOptions)
+	return Markdown(input, DefaultOptions)
 }
 
 // Markdown is the main rendering function.
 // It parses and renders a block of markdown-encoded text.
-// The supplied Renderer is used to format the output, and extensions dictates
-// which non-standard extensions are enabled.
+// The supplied options contain a Renderer used to format the output, and
+// extensions that dictate which non-standard extensions are enabled.
 //
-// To use the supplied HTML renderer, see NewHTMLRenderer.
-func Markdown(input []byte, renderer Renderer, options Options) []byte {
-	if renderer == nil {
-		return nil
+// If options.Renderer is nil, the supplied HTML renderer is used.
```

Comment:
Oh yes, by "supplied" I meant exactly that, the default one. That's an obviously poor choice of words. I like your suggestion to be explicit about what's used instead of nil.